package exceptioninJava;

public class TryTest {

	public static void main(String[] args) {
		int arr[]={2,4,5,6,9};
		System.out.println(arr[2]+arr[4]);

	}

}
